concepts/storage-class-specifiers/introduction.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+Storage-class specifiers relate to how variables are stored in memory.
+They are closely related to the storage duration (also referred to as the lifetime) of a value.
+
+## auto (the default storage class for function or block scope variables)
+
+Since variables defined within a block or function are `auto` by default, it is not common to explicitly use the term.
+An `auto` variable's lifetime begins when its block is entered and ends when its block is exited.
+An `auto` variable has memory allocated for it, _but with no default value_, when its block is entered.
+
+## static (the storage specifier not to be confused with the static linkage type)
```

Comment:
```suggestion
## static (the storage specifier not to be confused with the static linkage type)

```

Reply:
Unable to directly commit this suggestion as it's outdated by my correcting it while doing something else.